src/components/notes.js

```diff
@@ -0,0 +1,47 @@
+import "../App.css";
+import { useState } from "react";
+
+const Note = () => {
+  const [show, setShow] = useState(false);
+  const [note, setNote] = useState();
+  const [notesArray, setnotesArray] = useState([]);
+  const clickHandler = (e) => {
+    if (show !== true) {
+      setShow(true);
+    } else {
+      setShow(false);
+    }
+  };
+
+  const noteAdd = (e) => {
+    setNote("");
+    setnotesArray([...notesArray, note]);
+  };
+
+  return (
+    <>
+      {" "}
+      <button onClick={(e) => clickHandler(e)}>ToDo</button>
+      {show ? (
```

Comment:
If you need to satisfy only one condition use && instead of ternary oprerator.

Reply:
ok